azurerm/resource_arm_app_service.go

```diff
@@ -433,6 +458,11 @@ func resourceArmAppServiceRead(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("Error making Read request on AzureRM App Service Configuration %q: %+v", name, err)
 	}
 
+	authResp, err := client.GetAuthSettings(ctx, resGroup, name)
+	if err != nil {
+		return fmt.Errorf("Error making Read request on AzureRM App Service AuthSettings %q: %+v", name, err)
```

Comment:
could we alter the phrasing here to match newer resources:

```suggestion
		return fmt.Errorf("Error retrieving the AuthSettings for App Service %q (Resource Group %q): %+v", name,  resGroup, err)
```